src/features/languages/generated/client/client.json

```diff
@@ -185,7 +185,6 @@
         "config.form.tooltip.requiredForRegistration": "Select if this field is mandatory to complete registration",
         "config.form.tooltip.conditionalForRegistration": "Select if this field is conditional to complete registration",
         "config.form.tooltip.contentKey": "Use these keys to update copy for this field",
-        "config.form.tooltip.fieldId": "Use these keys to search for specific field items",
```

Comment:
@CornelMaze I see quite a few translations were removed in this PR? Were they intentional?